fineract-provider/src/main/java/org/apache/fineract/portfolio/account/api/AccountTransfersApiResourceSwagger.java

```diff
@@ -320,7 +320,7 @@ static final class GetAccountTransfersStatus {
                 private GetAccountTransfersStatus() {}
 
                 @Schema(example = "300")
-                public Integer id;
```

Comment:
This looks wrong to me...It should be Long based on the EnumOptionData line:36

Reply:
Done.